app/src/features/reports/TellMeAboutReport.tsx

```diff
@@ -93,48 +96,59 @@ function TellMeAboutReport(props: { variable: VariableId }) {
 
   return (
     <WithDatasets datasetIds={requiredDatasets}>
-      {() => (
-        <Grid container spacing={1} alignItems="flex-start">
-          <Grid item xs={12} sm={12} md={6}>
-            Filter results by race:
-            <FormControl>
-              <Select
-                name="raceSelect"
-                value={race}
-                onChange={(e) => {
-                  setRace(e.target.value as string);
-                  setCountyList([]);
-                }}
-              >
-                {RACES.map((race) => (
-                  <MenuItem key={race} value={race}>
-                    {race}
-                  </MenuItem>
-                ))}
-              </Select>
-            </FormControl>
-            <UsaChloroplethMap
-              signalListeners={signalListeners}
-              varField={props.variable}
-              legendTitle={variableProvider.variableName}
-              filterVar="race"
-              filterValue={race}
-              data={variableProvider.getData(
-                datasetStore.datasets,
-                Breakdowns.byState().andRace()
-              )}
-              operation="sum"
-            />
-          </Grid>
-          <Grid item xs={12} sm={12} md={6} className={styles.PaddedGrid}>
-            <p>
-              Click on some states to see data in this table, shift click on map
-              to reset.
-            </p>
-            {CountyLevelTable(countyList, variableProvider.variableName)}
+      {() => {
+        const breakdowns =
+          race === THE_USA_STRING
+            ? Breakdowns.byState()
+            : Breakdowns.byState().andRace();
+        let dataset = variableProvider
+          .getData(datasetStore.datasets, breakdowns)
+          .filter((r) => race === THE_USA_STRING || r.race === race);
```

Comment:
Why is this filtering using an OR? Shouldn't it just be r.race === race and rely on the race state?